src/verify.c

```diff
@@ -253,11 +253,14 @@ static int get_required_files(struct manifest *official_manifest, struct list *s
 		progress_set_next_step("download_packs");
 		print("\n");
 		get_all_files(official_manifest, subs);
+		progress_complete_step();
 	}
 
 	progress_set_next_step("check_files_hash");
 	print("\n");
 	if (check_files_hash(official_manifest->files)) {
+		progress_set_next_step("");
```

Comment:
I don't get this, shy are you setting an empty step here?

Reply:
For consistency. To keep the number of steps constant